Integrations/class.duplicate-post.php

```diff
@@ -3,21 +3,24 @@
  * Integrate with Duplicate Post plugin in order to copy
  * Mesh content when a post is created.
  *
- * https://duplicate-post.lopo.it/docs/developers-guide/actions/dp_duplicate_post/
+ * Plugin https://duplicate-post.lopo.it/docs/developers-guide/actions/dp_duplicate_post/ url.
```

Comment:
I know this is a dumb thing where comments have to end with a "hard stop" and I didn't want to add a . to the end of a url.